### ISSUE TITLE
Pagination missing accessible name for landmark

### ISSUE THREAD

**Issue by jaredcunha** [department-of-veterans-affairs/vets-design-system-documentation#1332]
# Bug Report

- [x] I’ve searched for any related issues and avoided creating a duplicate issue.

## What happened

Because the pagination component is a navigation landmark without an accessible name, it's hard to distinguish it from form the main navigation in the rotor menu.

<img width="926" alt="Screen Shot 2022-11-10 at 2 09 58 PM" src="https://user-images.githubusercontent.com/1094951/201185316-afd26d10-9476-4492-bde0-30db56e1b6d5.png">

## What I expected to happen

Using `aria-label="Pagination"` will help screen reader users distinguish the pagination navigation landmark from the main navigation. 

## Reproducing
- Formation version: current
- Device: (e.g. iPhone 8, Macbook): any
- Browser: (e.g. Firefox, IE 11): any

Steps to reproduce:

1. Log into staging.va.gov with user `+228`
2. Visit any page with `va-pagination`, for example, https://staging.va.gov/track-claims/your-claim-letters
3. Turn on VoiceOver and open Landmarks menu
4. Note there are two items 

## Urgency
How urgent is this request? Please select the approriate option below and/or provide details
- [ ] This bug is blocking work currently in progress
- [ ] This bug is affecting work currently in progress but we have a workaround
- [ ] This bug is blocking work planned within the next few sprints
- [x] This bug is not blocking any work
- [ ] Other

### Details

Possible resolution would be to add `aria-label="Pagination"` to [this line.](https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-pagination/va-pagination.tsx#L163)
